src/notifications/common.ts

```diff
@@ -55,6 +57,7 @@ export const notificationPreferenceMap: Partial<
   [NotificationType.SquadReply]: NotificationPreferenceType.Comment,
   [NotificationType.SquadPostAdded]: NotificationPreferenceType.Source,
   [NotificationType.SquadMemberJoined]: NotificationPreferenceType.Source,
+  [NotificationType.CollectionUpdated]: NotificationPreferenceType.Post,
```

Comment:
This should be `post`? (not sure if it should be it's own type at this point)

Reply:
Actually do think this is right, just to confirm by @capJavert

Reply:
looks good